cmd/cosign/cli/options/root.go

```diff
@@ -43,4 +44,7 @@ func (o *RootOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.PersistentFlags().DurationVarP(&o.Timeout, "timeout", "t", DefaultTimeout,
 		"timeout for commands")
+
+	cmd.PersistentFlags().BoolVarP(&o.SkipConfirmation, "yes", "y", false,
+		"skip confirmation prompts")
```

Comment:
Can we provide more detail about what it does? "skip confirmations prompts for non-destructive actions" or similar?